spec/unleash/feature_toggle_spec.rb

```diff
@@ -20,16 +20,126 @@
 
   describe 'FeatureToggle with empty strategies' do
     let(:feature_toggle) { Unleash::FeatureToggle.new(
-      name: 'test',
-      enabled: true,
-      strategies: [],
-      variants: nil
+      JSON.parse('{
+          "name": "test",
+          "enabled": true,
+          "strategies": [],
+          "variants": []
+        }')
       ) }
 
-    it 'should return true' do
+    it 'should return true if enabled, and default is true' do
       context = Unleash::Context.new(user_id: 1)
       expect(feature_toggle.is_enabled?(context, true)).to be_truthy
     end
+
+    it 'should return true if enabled, and default is false' do
+      context = Unleash::Context.new(user_id: 1)
+      expect(feature_toggle.is_enabled?(context, false)).to be_truthy
+    end
+  end
+
+  describe 'FeatureToggle with empty strategies and disabled toggle' do
+    let(:feature_toggle) { Unleash::FeatureToggle.new(
```

Comment:
Again, I would prefer if the parameters were sent to the constructor as a `Hash` instead of having to go through the extra step of de-serialization.

Reply:
fix on commit 1ca04ac72fd6edcd507860350b28d74d2172bb9f